nibabel/cifti2/__init__.py

```diff
@@ -6,14 +6,15 @@
 #   copyright and license terms.
 #
 ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
-"""CIfTI format IO
+"""CIfTI2 format IO
```

Comment:
Silly pedantic thing, but CIFTI (like GIFTI) doesn't have a lowercase `f`. Would you mind doing a find-and-replace for this? (The `Cifti2` that refers to the prefix of, e.g. `Cifti2Image` or `Cifti2BrainModel` doesn't need updating. Just the format itself.)

```suggestion
"""CIFTI-2 format IO
```